### PR TITLE
mklive.sh: allow passing multiple include dirs (-I)

### PR DESCRIPTION
**Pull request by paper42**
suggested in https://github.com/void-linux/void-mklive/pull/300#issuecomment-1439119162 by @0x5c

### CODE REVIEW

File: mklive.sh.in [hunk 1]
```diff
@@ -303,7 +305,7 @@ while getopts "a:b:r:c:C:T:Kk:l:i:I:S:s:o:p:v:h" opt; do
         k) KEYMAP="$OPTARG";;
         l) LOCALE="$OPTARG";;
         i) INITRAMFS_COMPRESSION="$OPTARG";;
-        I) INCLUDE_DIRECTORY="$OPTARG";;
+        I) INCLUDE_DIRS="$INCLUDE_DIRS $OPTARG";;
```
Comment (0x5c) [inline mklive.sh.in:308]: While I really dislike paths that contain spaces, they do happen somewhat frequently and I think they should probably be handled here.
The separator should perhaps be something that cannot be in a filename?

File: mklive.sh.in [hunk 2]
```diff
@@ -417,9 +419,9 @@ install_packages
 print_step "Enabling services: ${SERVICE_LIST} ..."
 enable_services ${DEFAULT_SERVICE_LIST} ${SERVICE_LIST}
 
-if [ -n "${INCLUDE_DIRECTORY}" ];then
-    print_step "Copying directory structure into the rootfs: ${INCLUDE_DIRECTORY} ..."
-    copy_include_directory
+if [ -n "${INCLUDE_DIRS}" ];then
+    print_step "Copying directory structures into the rootfs: ${INCLUDE_DIRS} ..."
```
Comment (0x5c) [inline mklive.sh.in:423]: The list of include directories can get a bit busy in the output, maybe they could be printed individually using `info_msg()` in the for loop of `copy_include_directories()` instead?

**Comment by 0x5c** [comment]
Got to test an image now, the functionality works

File: mklive.sh.in (hunk 1 above)
Comment (classabbyamp) [reply mklive.sh.in:308]: IMO just make this script bash and use an array

**Comment by classabbyamp** [comment]
I've converted this to bash arrays to make it easier to handle, and moved the message showing the include dir names to inside the loop

**Pull request merged**